src/monika/Assignment14/CalculatorPractise.java

```diff
@@ -0,0 +1,42 @@
+package monika.Assignment14;
+
+import java.util.Scanner;
+
+public class CalculatorPractise {
+	 Scanner sc = new Scanner(System.in);
+	
+	int add(int num1, int num2) {
+		return num1+num2;
+	}
+	
+	int sub(int num1) {
```

Comment:
this is not expected, 
think on this for calculator we pass two value in method arguments but here is the only one and second argument you have take from scanner class this is fine but service method not take input from user, service method take input and perform action.